test/unity/toolkits/object_detection/one_shot_object_detection/test_parameter_sampler.cxx

```diff
@@ -0,0 +1,105 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define BOOST_TEST_MODULE test_parameter_sampler
+
+#include <toolkits/object_detection/one_shot_object_detection/util/parameter_sampler.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include <core/util/test_macros.hpp>
+
+#include <unordered_map>
+
+static constexpr int max_dimension = 6000;
```

Comment:
Let's give constants a distinctive syntax, either `MAX_DIMENSION` or `k_max_dimension`

(FWIW, https://google.github.io/styleguide/cppguide.html#Constant_Names, although it'd be a pretty big change for us to adopt Google's CamelCase for C++ names in the general case)

Reply:
Done. Followed the Google style guide.